app/views/wishlists/show.html.erb

```diff
@@ -1,14 +1,29 @@
-<h1><%= @wishlist.name %></h1>
+<div class="container">
+  <div class="text-center">
+    <h1><%= @wishlist.name %></h1>
 
-<p><%= @wishlist.description %></p>
-
-<p>
-  <strong>Your unique link for this whislist:</strong><br>
-  <%= wishlists_for_guest_url(@wishlist.token) %>
-</p>
-
-<h2>Gifts:</h2>
-  <ul>
+    <p><%= @wishlist.description %></p>
+    <p>
+      <div class="input-group col-xs-6 col-xs-offset-3">
```

Comment:
Read up on how to use the grid system. You're missing a `<div class="row">...</div>` wrapper. And `div`s with those classed should be wrappers, and not be wrapped by a `<p>` tag.